src/SoundBank.js

```diff
@@ -0,0 +1,84 @@
+const SoundPlayer = require('./GreenPlayer');
+const EffectsChain = require('./EffectsChain');
+
+const ALL_TARGETS = '*';
+
+class SoundBank {
+    constructor (audioEngine) {
+        this.audioEngine = audioEngine;
+
+        this.soundPlayers = {};
+        this.playerTargets = new Map();
+        this.soundEffects = new Map();
+    }
+
+    getSound (soundId) {
+        if (!this.soundPlayers[soundId]) {
+            this.soundPlayers[soundId] = new SoundPlayer(this.audioEngine, {
+                id: soundId, buffer: this.audioEngine.audioBuffers[soundId]
+            });
+        }
+
+        return this.soundPlayers[soundId];
+    }
+
+    getSoundEffects (sound) {
+        if (!this.soundEffects.has(sound)) {
+            this.soundEffects.set(sound, new EffectsChain(this.audioEngine));
```

Comment:
I think it'd be better to have a `this.effectsStandard.clone()` way to make new EffectsChains.